project-base/build.xml

```diff
@@ -89,6 +89,13 @@
         </delete>
     </target>
 
+    <target name="clean-redis" description="Cleans up redis cache">
```

Comment:
People would be confused that `clean` command deletes only var folder. What about create command `clean-all` or use `clean` (and rename old `clean` to `clean-var`) which will clean var folder and redis cache together